core/templates/dev/head/services/explorationContextServiceSpec.js

```diff
@@ -88,7 +88,7 @@ describe('Exploration context service', function() {
     });
 
     beforeEach(inject(function($injector) {
-      ecs = $injector.get('explorationContextService');
+      ecs = $injector.get('ExplorationContextService');
     }));
 
     it('should throw an error when trying to retrieve the exploration id',
```

Comment:
Here too, let's have the file renamed from `explorationContextServiceSpec.js` to `ExplorationContextServiceSpec.js`.